pkg/libursa/ursa/ursa_test.go

```diff
@@ -47,4 +47,12 @@ func TestBlindedCredentialSecretsCorrectnessProofFromJSON(t *testing.T) {
 		assert.NotEmpty(t, err)
 		assert.Empty(t, correctnessProof)
 	})
+}
+
+func TestCredentialPublicKeyFromJSON(t *testing.T) {
+	t.Run("CredentialPublicKeyFromJSON", func(t *testing.T) {
+		credPubKey, err := CredentialPublicKeyFromJSON("should error")
+		assert.NotEmpty(t, err)
+		assert.Empty(t, credPubKey)
+	})
 }
```

Comment:
You need to add a positive test case here to prove that with a valid JSON, it will work.